zcash_client_backend/src/zip321.rs

```diff
@@ -426,8 +489,22 @@ mod parse {
         Other(String, String),
     }
 
+    impl Param {
+        /// Returns the name of the parameter from which this value was parsed.
+        pub fn name(&self) -> String {
```

Comment:
Changelog addition (or make it crate-private if you only want to use it for nice errors).